test/ToastSpec.js

```diff
@@ -45,4 +45,14 @@ describe('Toasts', () => {
     clock.tick(500);
     expect(onCloseSpy).to.be.calledOnce;
   });
+
+  it('should clearTimeout after unmount', () => {
```

Comment:
should this assert something?

Reply:
not sure if it makes sense. But it will cover that: https://github.com/react-bootstrap/react-bootstrap/blob/master/src/Toast.js#L77-L79

Do you think I can check if the timeout was cleared? hmm

Reply:
could you manually trigger timers with jest, but check that `onClose` is not called?

Reply:
sinon, rather, not jest

Reply:
currently we won't fire `onClose` on `componentWillUnmount` not sure if this makes sense.

In the Modal it will be thrown: https://github.com/react-bootstrap/react-overlays/blob/master/src/Modal.js#L257-L259

Reply:
i mean, that `onClose` does _not_ get fired is a good way to ensure that the timeout gets cleared, right?

if the timeout _didn't_ get cleared, then `onClose` might get called

Reply:
can you add a fake clock here and make it tick, as with the above?